contrib/virt-manager/patches/no-gtk-update-icon-cache.patch

```diff
@@ -0,0 +1,22 @@
+diff --git a/setup.py b/setup.py
+index b45d315..48d6e21 100755
+--- a/setup.py
++++ b/setup.py
+@@ -266,17 +266,6 @@ class my_install(setuptools.command.install.install):
+     def run(self):
+         setuptools.command.install.install.run(self)
+ 
+-        if not self.distribution.no_update_icon_cache:
```

Comment:
[Void avoids this patch](https://github.com/void-linux/void-packages/blob/master/srcpkgs/virt-manager/template) by passing `--no-update-icon-cache --no-compile-schemas` to `setup.py`

Reply:
we can't pass args to setup.py because the pep517 standard (invoked via python -m build) does not have a way to pass standard args yet..

Reply:
so the patch is fine, it's just a few lines

Reply:
sad, but fair enough